App programmering/Opgave/app/src/main/java/com/example/app/MainActivity.java

```diff
@@ -10,4 +10,110 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
     }
+
+    private float si2local_volume (int volume, float val) {
+        if (val == 0.0) {
+            return (float) 0.00;
+        }
```

Comment:
Jeg forstår ikke helt hvad dette gør godt for. Kunne du eventuelt lave en kommentar?

Reply:
Den skal sørge for at der er 2 decimaler i det tekstfelt den bliver indsat i, selvom tallet er lig 0 eller tomt. Dette er fordi tanken er at et "actual" flow, godt kan være 0, og for at det ser pænt ud med de andre beregninger af bl.a. pulse output osv som alle har 2 decimaler vist på skærmen. Så det er meget simpelt kun derfor.

Reply:
Det tror jeg ikke du kan gøre på den måde. Har du testet efter at det virker?